cni/scripts/cleanupnetwork.ps1

```diff
@@ -0,0 +1,76 @@
+$Global:ClusterConfiguration = ConvertFrom-Json ((Get-Content "c:\k\kubeclusterconfig.json" -ErrorAction Stop) | out-string)
+
+$global:NetworkMode = "L2Bridge"
+$global:ContainerRuntime = $Global:ClusterConfiguration.Cri.Name
+$global:NetworkPlugin = $Global:ClusterConfiguration.Cni.Name
+$global:HNSModule = "c:\k\hns.psm1"
```

Comment:
also we can get this from their github repo directly instead of relying on local disk